### PR TITLE
site updates for 4.15

### PR DESCRIPTION
**Pull request by DaanHoogland**

### CODE REVIEW

**Comment by andrijapanicsb** [comment]
"archives" should probably have 4.15 branch and 4.15.0.0 added, if the current one (4.14) looks OK. LGTM otherwise.

**Review by rhtyd — APPROVED** [review]

File: data/cloudstack.yml [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-  lts: 4.14.0.0
+  main_docs_url: http://docs.cloudstack.apache.org/en/4.15.0.0/
+  branch: 4.15
+  lts: 4.15.0.0
   lts_docs_url: http://docs.cloudstack.apache.org/en/latest/installguide/index.html
-  ltsbranch: 4.14
+  ltsbranch: 4.15
   archives:
     - '4.14':
        - 4.14.0.0
```
Comment (rhtyd) [inline data/cloudstack.yml:11]: @DaanHoogland can you add 4.14.1.0 here as well. It seem the master branch is not updated, but asf-site is.

Comment (DaanHoogland) [reply data/cloudstack.yml:13]: not sure what the semantics are, can you check if this is what you mean @rhtyd ? (I'll run middleman as well to check)

**Comment by DaanHoogland** [comment]
Just checked, this PR is very stale, closing

**Pull request closed without merging**